### PR TITLE
GDB-5576 Fixed in my-settings.spec.js the way how checkbox value is taken.

### PR DESCRIPTION
**Pull request by sava-savov-ontotext**
Also flaky restart repository test, where new library implementing waitUntil method is provided.

### CODE REVIEW

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

File: test-cypress/support/sparql-commands.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ Cypress.Commands.add('pasteQuery', (query) => {
     clearQuery();
     // Using force because the textarea is not visible
     getQueryTextArea().invoke('val', query).trigger('change', {force: true});
-    waitUntilQueryIsVisible();
+    waitUntilQueryIsVisible(query);
```
Comment (desislava-hristova-ontotext) [inline test-cypress/support/sparql-commands.js:5]: The function does not accept arguments. I guess this change is obsolete.

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Pull request merged**